code/game/objects/structures/aliens.dm

```diff
@@ -256,6 +250,7 @@
 	var/obj/structure/alien/wallweed/wall_weed // This var is used to handle wall-weed interactions for when they need to be deleted
 	var/static/list/weedImageCache
 	var/check_counter // This var is how many process() procs it takes for a weed to spread
+	var/silent_removal = FALSE // Needed for automatic removal
```

Comment:
can you make this autodoc and explain what this var does

Reply:
I think I did what was asked, do tell if its not enough

Reply:
Auto doc is done with 3 `/` and on the line above the var instead of next to it

Reply:
Ah I see, it should be good now!